test/plugins/templates.test.js

```diff
@@ -10,6 +10,7 @@ const testtemplate = require('./data/template.json');
 const testtemplates = require('./data/templates.json');
 const testtemplatetags = require('./data/templateTags.json');
 const testtemplateversions = require('./data/templateVersions.json');
+const testtemplateversionsmetrics = require('./data/templateVersionsMetrics.json');
```

Comment:
camelcase would be better
```suggestion
const testTemplateVersionsMetrics = require('./data/templateVersionsMetrics.json');
```

Reply:
updated it, thanks